resources/views/components/menu/management.blade.php

```diff
@@ -1,83 +1,87 @@
-            <div class="dropdown-divider"></div>
-        @endcan--}}
-        @can('manage', App\Platform\Models\IntegrationRelease::class)
-            <h6 class="dropdown-header">Releases</h6>
-            @can('manage', App\Platform\Models\Emulator::class)
-                <a class="dropdown-item {{ request()->routeIs('emulator*') ? 'active' : '' }}"
-                   href="{{ route('emulator.index') }}">
-                    Emulators
-                </a>
+        @if($user->Permissions >= Permissions::Developer)
+            <x-dropdown-header>{{ __('Community') }}</x-dropdown-header>
+            @can('manage', App\Community\Models\News::class)
+                {{--<x-dropdown-item :link="route('news.index')">{{ __res('news') }}</x-dropdown-item>--}}
+                <x-dropdown-item :link="url('submitnews.php')">{{ __res('news') }}</x-dropdown-item>
             @endcan
-            <a class="dropdown-item {{ request()->routeIs('integration.release*') ? 'active' : '' }}"
-               href="{{ route('integration.release.index') }}">
-                Integration
-            </a>
-            <div class="dropdown-divider"></div>
-        @endcan
+            @can('manage', App\Community\Models\Forum::class)
+                {{--<x-dropdown-item :link="route('forum-topic.clearing')">Forum Clearing</x-dropdown-item>--}}
+                <x-dropdown-item :link="url('viewforum.php?f=0')">Forum Clearing</x-dropdown-item>
+            @endcan
+            {{--@can('manage', App\Site\Models\Event::class)
+                <h6 class="dropdown-header">Events</h6>
+            @endcan--}}
+            {{--@can('manage', App\Platform\Models\IntegrationRelease::class)
+                <x-dropdown-header>Releases</x-dropdown-header>
+                @can('manage', App\Platform\Models\Emulator::class)
+                    <x-dropdown-item :link="route('emulator.index')" :active="request()->routeIs('emulator*')">Emulators</x-dropdown-item>
+                @endcan
+                <x-dropdown-item :link="route('integration.release.index')" :active="request()->routeIs('integration.release*')">Integration</x-dropdown-item>
+            @endcan--}}
+            @if($user->Permissions >= Permissions::Moderator)
+                <x-dropdown-item :link="url('admin.php')">Admin Tools</x-dropdown-item>
+            @endif
+        @endif
         @can('root')
-            <h6 class="dropdown-header">System Tools</h6>
+            <x-dropdown-header>{{ __('System') }}</x-dropdown-header>
             @can('viewHorizon')
                 <a class="dropdown-item" href="{{ url('horizon') }}" target="_blank">
                     Horizon Queue
                     <x-fas-external-link-alt/>
                 </a>
             @endcan
             @can('viewLogs')
-                <a class="dropdown-item" href="{{ url('log-viewer') }}" target="_blank">
+                <a class="dropdown-item" href="{{ route('log-viewer.index') }}" target="_blank">
```

Comment:
It's handled in the [AuthServiceProvider](https://github.com/RetroAchievements/RAWeb/blob/master/app/Site/AuthServiceProvider.php#L120):
```php
Gate::define('viewLogs', fn (Authenticatable $user) => $user->can('root'));
```

Because there's no explicit `root` gate defined it will default to the [`Gate::before()` closure](https://github.com/RetroAchievements/RAWeb/blob/master/app/Site/AuthServiceProvider.php#L42-L57)
```php
if ($user->Permissions >= Permissions::Root) {
  return true;
}
```

The root permission is not assigned to anyone but will be introduced again to make the permission matrix portable. This was not meant to be in this PR yet - reverted in https://github.com/RetroAchievements/RAWeb/pull/1776/commits/ce38b521388f9ecf270e85e31c2c736b5f1c944d